test/system/710-start.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats   -*- bats -*-
+
+load helpers
+
+function setup() {
+    basic_setup
+    run_podman run -d $IMAGE /bin/true
+    cid_none="$output"
+    run_podman run -d --restart=always $IMAGE /bin/true
+    cid_always="$output"
+}
+
+function teardown() {
```

Comment:
Please add `podman rm $cid_none $cid_always` here; this will reduce test-log clutter.